packages/react/src/components/Storage/FileUploader/UploadButton/__tests__/UploadButton.tsx

```diff
@@ -0,0 +1,43 @@
+import * as React from 'react';
```

Comment:
Can we rename this file to _UploadButton.test.tsx_?

Reply:
👍

Reply:
[885920e](https://github.com/aws-amplify/amplify-ui/pull/3028/commits/885920e203a8c6326d6086725c05a11f0b2e602f)